simapp.go

```diff
@@ -726,7 +726,7 @@ func UpdateConfig(f string) error {
 					var msg configMessage
 					msg.msgPtr = reqMsgBody
 					msg.msgType = subscriber
-					msg.name = strconv.FormatUint(k, 10)
+					msg.name = strconv.FormatUint(uint64(k), 10)
```

Comment:
will this not remove the 0's at the start of IMSI ? Can you check if message sent towards Webconsole has right URL.  msg.name is used to "            httpend = subscriberHttpend + msg.name"

Reply:
delete might not work.. let me check

Reply:
Thanks Ajay. Delete didn't work. Fixed it and pushed changes. Add /delete of subscribers worked with latest